src/Main.java

```diff
@@ -1,5 +1,263 @@
+        operator = input.next().charAt(0);
+        System.out.print("Enter first number : ");
+        a=input.nextInt();
+        System.out.print("Enter second number : ");
+        b=input.nextInt();
+        double addition  = a+b;
+        double subtraction  = a-b;
+        double multiplication  = a*b;
+        double division  = a/b;
+
+        if (operator == '+')
+        {
+            System.out.print("Total after Addition is : " + addition);
+        }    else if(operator == '-')
+        {
+            System.out.print("Total after Subtraction is : " +subtraction);
+        }
+        else if(operator == '*')
+        {
+            System.out.print("Total after Multiplication is : "+multiplication);
+        }
+        else if(operator == '/')
+        {
+            System.out.print("Total after Division is : "+division);
+        } else
+        {
+            System.out.print("Please select proper operator");
+        }
+
+        */
+        //Zadatak 1.f
+
+        int numberTaskF = 0;
+        while (true) {
+            System.out.println("Which operation would you like to use: *,/,+,-,= ?");
+            char operation = input.next().charAt(0);
+            if (operation != '+' && operation != '-' && operation != '*' && operation != '/' && operation != '=') {
+                System.out.println("Please select proper operator.");
+                continue;
+            } else if (operation == '=') {
+                if (numberTaskF == 0) {
+                    System.out.println("You have not entered any values");
+                } else {
+                    System.out.println("Total is " + numberTaskF);
+                }
+                break;
+            }
+            System.out.println("Please enter integer");
+            int numberYouEntered = input.nextInt();
+            if (numberYouEntered == -1000) {
+                System.out.println("Exit number -1000 has been detected");
+                break;
+            } else if (operation == '+') {
+                numberTaskF = numberTaskF + numberYouEntered;
+            } else if (operation == '-') {
+                numberTaskF = numberTaskF + numberYouEntered;
+            } else if (operation == '*') {
+                numberTaskF = numberTaskF * numberYouEntered;
+            } else if (operation == '/') {
+                numberTaskF = numberTaskF/ numberYouEntered;
+            }
+        }
+    }
+
+
+
+
+
+    //  Zadatak 2
+    // Napravite petlju u koju unosite brojeve kada petlja detektuje broj -10, ispise vam se niz brojeva koje ste unjeli.
+    // Zatim ce vam se na ekranu pojaviti pitanje da li zelite da prosirite array sa opcijama y/n.
+    //Ukoliko upisete y, ponovno unosite brojeve koji cete dodavati na niz kada ponovno unesete -10 ispisu vam se svi brojevi koje ste unjeli.
+
+       /*
+        Scanner input = new Scanner(System.in);
+        int[] array = new int[10];
+        int i = 0, temporary,counter=1;
```

Comment:
we didnt use this type of declaring variables, it is hard to read, and hard to see